synapse/models/person.py

```diff
@@ -490,13 +490,20 @@ def getModelDefs(self):
                     ('crypto:address', ('crypto:currency:address', {}), {
                         'doc': 'A crypto currency address associated with the contact.'
                     }),
-
                     ('lang', ('lang:language', {}), {
                         'alts': ('langs',),
                         'doc': 'The language specified for the contact.'}),
-
                     ('langs', ('array', {'type': 'lang:language'}), {
                         'doc': 'An array of alternative languages specified for the contact.'}),
+                    ('banner', ('file:bytes', {}), {
```

Comment:
Isn't the banner usually a truncated portion of the response vs the whole response? file:bytes feels heavier than needed here.

Reply:
This would be like an image on your account, and mirrors inet:web:acct:banner.  I'll defer to @invisig0th if we want to tweak this, or maybe add more context to the doc string?

Reply:
Oh, right! I remember chatting about it now. I was thinking about the *other* type of banner. Nm then :)